### PR TITLE
REF: DataFrame.mask tests

### PR DESCRIPTION
**Pull request by jbrockmendel**
we already did this for Series.mask

### CODE REVIEW

**Review by mroeschke — APPROVED** [review]

File: pandas/tests/frame/indexing/test_mask.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
+"""
+Tests for DataFrame.mask; tests DataFrame.where as a side-effect.
```
Comment (simonjayhawkins) [inline pandas/tests/frame/indexing/test_mask.py:2]: why is this not tests/frame/methods/test_mask and tests/frame/methods/test_where?

Comment (jreback) [reply pandas/tests/frame/indexing/test_mask.py:2]: where/mask are indexing methods

Comment (simonjayhawkins) [reply pandas/tests/frame/indexing/test_mask.py:2]: I don't understand the criteria for this directory, test_at_time, test_asof, test_between_time and other similar methods are in tests/frame/methods/test_*

Comment (jbrockmendel) [reply pandas/tests/frame/indexing/test_mask.py:2]: thats a fair question.  ill add a docstring to the directory `__init__` to try to clarify

**Pull request merged**